v1_20/src/main/java/com/denizenscript/denizen/nms/v1_20/impl/network/handlers/DenizenNetworkManagerImpl.java

```diff
@@ -49,11 +49,13 @@
 import java.util.stream.Collectors;
 
 public class DenizenNetworkManagerImpl extends Connection {
+    // TODO: 1.20.6: there are some new methods that should potentially be overriden

```

Comment:
Yes, override all new methods with redirect to oldManager, there's only a select few exceptions to the mandatory override